src/game/rtsutils.h

```diff
@@ -58,14 +59,14 @@ template<typename T> struct less_than_nocase
     bool operator()(const T &left, const T &right) const { return (left.Compare_No_Case(right) < 0); }
 };
 
-template<typename T> struct equal_to
+template<typename T> struct hash
 {
-    bool operator()(const T &left, const T &right) const { return (left.Compare(right) == 0); }
+    size_t operator()(const T &object) const { return static_cast<size_t>(object); }
```

Comment:
Is this confirmed what original does?

Reply:
It copies the stl version. This class is just a duplicate of the stl version but modified so that it can take a utf8string

Reply:
Ok that is fine then.